src/graph/test/TestEnv.cpp

```diff
@@ -39,15 +39,28 @@ void TestEnv::SetUp() {
     auto addrsRet
         = network::NetworkUtils::toHosts(folly::stringPrintf("127.0.0.1:%d", metaServerPort()));
     CHECK(addrsRet.ok()) << addrsRet.status();
-    mClient_ = std::make_unique<meta::MetaClient>(threadPool, std::move(addrsRet.value()), true);
-    mClient_->init();
+    auto storagePort = network::NetworkUtils::getAvailablePort();
+    auto hostRet = nebula::network::NetworkUtils::toHostAddr("127.0.0.1", storagePort);
+    if (!hostRet.ok()) {
+        LOG(ERROR) << "Bad local host addr, status:" << hostRet.status();
+    }
+    auto& localhost = hostRet.value();
+    mClient_ = std::make_unique<meta::MetaClient>(threadPool,
+                                                  std::move(addrsRet.value()),
+                                                  localhost,
+                                                  true);
+    auto r = mClient_->addHosts({localhost}).get();
```

Comment:
Why we add a host and remove it later? waitForMetadReady need it for what?

Reply:
If we do not remove it, later it will affect the next test. Some test case request there is no host!